nixos/modules/services/monitoring/icinga2.nix

```diff
@@ -0,0 +1,77 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.icinga2;
+  environment = pkgs.writeText "icinga-env" ''
+    DAEMON=${pkgs.icinga2}/sbin/icinga2
+    ICINGA2_CONFIG_FILE=/etc/icinga2/icinga2.conf
```

Comment:
Any reason we "pollute" /etc? Can't we write the config to the nix store?

Reply:
merely debugging trying to get the thing to run in the first place. My end goal is to have it be a src the user can define as say a git repository or tarball somewhere.